adapters/interactiveOffers/interactiveOffers.go

```diff
@@ -0,0 +1,79 @@
+package interactiveOffers
```

Comment:
Package names in Go are preferred to be lower case. Please consider naming your package and folder `interactiveoffers`.